tapchannel/allocation.go

```diff
@@ -447,6 +453,7 @@ func DistributeCoins(inputs []*proof.Proof, allocations []*Allocation,
 				AnchorOutputInternalKey:      a.InternalKey,
 				AnchorOutputTapscriptSibling: sibling,
 				ScriptKey:                    a.ScriptKey,
+				ProofDeliveryAddress:         deliveryAddr,
```

Comment:
> In order for us to send the proofs to the correct proof courier address,
we need to be able to specify different delivery addresses for each
allocation.

In the future, or today?

Reply:
Today. Otherwise if the initiator of the shutdown process uses the non-default proof courier address, the respondent wouldn't ever learn that and wouldn't know where to download the proof from.